src/components/ui/command.tsx

```diff
@@ -0,0 +1,171 @@
+import * as React from "react"
+import { Command as CommandPrimitive } from "cmdk"
```

Comment:
Discovering these new packages along the way with the shadcn-ui approach. Loving it.